tests/test_redditdownload.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+"""test redditdownload module."""
+import json
+import os
+import sys
+import unittest
+
+
+import pytest
+try:  # py3
+    from mock import patch, MagicMock
+except ImportError:  # py2
+    from unittest import mock
```

Comment:
Note that in this case you don't seem to have a `patch` function which is used later.